keyboards/dztech/dz65rgbv3/readme.md

```diff
@@ -0,0 +1,13 @@
+# DZ65RGBv3
+
+A customizable 65% RGB keyboard.
+
+* Keyboard Maintainer: [moyi4681](https://github.com/moyi4681)
+* Hardware Supported: KBDFANS
+* Hardware Availability: [kbdfans](https://kbdfans.myshopify.com/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter and use the bootloader to flash new firmware?